### ISSUE TITLE
Minutes publishing reminder

### ISSUE THREAD

**Issue by frcroth** [fsr-de/myHPI#162]
Send a reminder to publish minutes after a certain number of days (See https://github.com/fsr-de/1327/issues/119)

**Comment by frcroth** [comment]
What is the current workflow in the FSR? Create a draft page or publish minutes and not assign any groups as visible for?

**Comment by abc013** [comment]
The current workflow is
- Minute gets written into a draft page
- We then iterate over it using another tool, e.g. Google Docs or NextCloud
- The result gets pasted into the draft. We then assign groups and publish the minute.

**Comment by frcroth** [comment]
@abc013 So would it be useful to have a reminder for draft minutes that have not been published for a certain number of days?

**Comment by jeriox** [comment]
Should the reminder also protect against the use case of #472, aka minutes that have been published but are not visible to any group?
I think this relates to the question whether the reminder should work like the one in 1327 (a single mail for a specific minutes if it is still a draft after x days) or more like a regular digest (one email every x days with a list of all draft minutes). I think the latter could easily include a section for those "fake-published" minutes.

**Comment by jeriox** [comment]
> This raises the question, however, on how to check the actual groups the minutes should be visible to. We could set a preference in the protocol list (which means, the parent page) and maybe have a checkbox or something for pages which should explicitely not follow the preferences to not get reminded of those the whole time.

good point. I'm afraid that this is too complicated, especially for non-FSR groups that are not as deep into the myHPI game as you are. Maybe it would just suffice to check if a minutes is visible to any group?

**Comment by abc013** [comment]
> Maybe it would just suffice to check if a minutes is visible to any group?
That should do the job as well. It won't catch minutes which were published to the wrong subset of groups, but that's a thing for #472.

**Comment by jeriox** [comment]
although that plan could collide with #328
